tests/mock_connection.py

```diff
@@ -61,7 +72,20 @@ def request(self,
+            elif body['groupId'] == '[-hub-][field-group][[[ems-core][entity-type][foqa-flights]][[source-flight-info][internal-field-group][source-system-information]]]':
+                resp_h = [('Cache-Control', 'no-store, must-revalidate, no-cache'), ('Pragma', 'no-cache'), ('Content-Type', 'application/json; charset=utf-8'), ('Content-Encoding', 'gzip'), ('Vary', 'Accept-Encoding'), ('Server', 'Microsoft-IIS/8.5'), ('X-Adi-Unique-Id', '64272b3a-f862-4ade-8252-229749bf7c8b'), ('X-Powered-By', 'ARR/3.0'), ('Date', 'Thu, 09 Jul 2020 16:51:20 GMT'), ('Connection', 'close'), ('Content-Length', '450')]
+                content = {'id': '[-hub-][field-group][[[ems-core][entity-type][foqa-flights]][[source-flight-info][internal-field-group][source-system-information]]]', 'name': 'Source System Information', 'groups': [], 'fields': [{'id': '[-hub-][field][[[ems-core][entity-type][foqa-flights]][[source-flight-info][base-field][from-source-ems]]]', 'type': 'boolean', 'name': 'From Source EMS'}, {'id': '[-hub-][field][[[ems-core][entity-type][foqa-flights]][[source-flight-info][base-field][source-ems-installation]]]', 'type': 'discrete', 'name': 'Source EMS Installation'}, {'id': '[-hub-][field][[[ems-core][entity-type][foqa-flights]][[source-flight-info][base-field][source-flight-record]]]', 'type': 'number', 'name': 'Source Flight Record'}, {'id': '[-hub-][field][[[ems-core][entity-type][foqa-flights]][[source-flight-info][base-field][source-tail-number]]]', 'type': 'string', 'name': 'Source Tail Number'}, {'id': '[-hub-][field][[[ems-core][entity-type][foqa-flights]][[source-flight-info][base-field][source-fleet-id]]]', 'type': 'number', 'name': 'Source Fleet ID'}, {'id': '[-hub-][field][[[ems-core][entity-type][foqa-flights]][[source-flight-info][base-field][source-download-record-creation-date]]]', 'type': 'dateTime', 'name': 'Source Download Record Creation Date'}]}
+            return resp_h, content
```

Comment:
What's this?

Reply:
Mock responses for the `MockFilterQuery` class

Reply:
> Mock responses for the `MockFilterQuery` class

Also missed that this one was a mock. Great!